### PR TITLE
docs(v11): color/overview updates

### PR DESCRIPTION
**Pull request by aagonzales**
Updates to the v11 color/overview tab based on feedback and new features.

#### Changelog

**New**

- H2s for easier navigation 
- New tabbed images in the Light and Dark themes sections

**Changed**

- Updated images for color anatomy to include a usage image along with the model illustration. 
- Updated color tokens section content, added a token naming, token group and component token section. 
- Updated alt text in images per Mike G suggestions
- Updated hover color section with new method. 

**Removed**

- Removed the "Applying color" intro section so we could elevate H3s to H2s for easier navigation.
- Removed color role section, its been rolled up into the token section.

### CODE REVIEW

File: src/pages/guidelines/color/overview.mdx [hunk 1]
```diff
@@ -43,7 +45,7 @@ library in the context of dark and light themes.
 
 <GifPlayer color="dark">
 
-![UI color alternatives](images/introduction.gif)
+![6 screenshots showing a range of color schemes and considerations for text and data visualizations](images/introduction.gif)
 ![UI color alternatives](images/introduction.jpg)
```
Comment (alisonjoseph) [inline src/pages/guidelines/color/overview.mdx:49]: This image seems to be missing? When I pause the gif there's a missing image on the page.

**Review by alisonjoseph — CHANGES_REQUESTED** [review]
noticed a missing image, otherwise this looks great!

File: src/pages/guidelines/color/overview.mdx (hunk 1 above)
Comment (aagonzales) [reply src/pages/guidelines/color/overview.mdx]: oh weird, source was `.jpeg` not `jpg`

**Review by jnm2377 — APPROVED** [review]

**Review by alisonjoseph — APPROVED** [review]

**Comment by aagonzales** [comment]
Reviewed this with the design team in our Thursday crit, made changes accordingly.
- Add back in "implementing color" section
- Fix Gray 90 theme image
- Consolidate some of the token content. 
- Fix high contrast image: Sub pop-over for tooltip
- Add more elements to the theme example images.

**Pull request merged**